src/webhook/handlers/tutorial.ts

```diff
@@ -406,5 +423,5 @@ export default function tutorial(params) {
   });
   visitor.send();
 
-  return { data, event, issuedAt, userId, replies };
+  return { data, event, userId, replies, state };
```

Comment:
nit: actually `state` is not needed; it is also excluded from `ChatbotStateHandlerReturnType`.

Since the user can scroll back in the conversation and click buttons to jump to arbitrary previous states, it is actually not very useful to store a `state` enum in the context. That is why we excluded `state` from handler's return type, and determine what the current state is inside `singleUserHandler` with action payload instead.

Reply:
ok then we will need this
```
let params: ChatbotStateHandlerParams | ChatbotStateHandlerReturnType = {
```